extension/resources/walkthrough/plots.md

```diff
@@ -1,61 +1,82 @@
 # Plots
 
-Select one or more experiments to visualize them in the
-[`Plots Dashboard`](command:dvc.showPlots). This is the extension's equivalent
-of the `dvc plots show` and `dvc plots diff` commands.
+Select one or more experiments to visualize in the
+[**Plots Dashboard**](command:dvc.showPlots).
 
-💡 If you don't have any DVC
-[plots](https://dvc.org/doc/command-reference/plots) in the project, start
-writing data points into JSON, YAML, CSV or TSV or saving plots as images
-(`.png`, etc) (check the [DVCLive](https://dvc.org/doc/dvclive) helper library
-if you use Python):
+> This is the similar to the [`dvc plots show`] and [`dvc plots diff`] commands.
```

Comment:
I liked the previous variation better (equivalent vs similar) - what is the reason changing this?

Reply:
Equivalent is more accurate now that I think about it. Committing:
```suggestion
> This is equivalent to the [`dvc plots show`] and [`dvc plots diff`] commands.
```